### PR TITLE
formula for atlassian-confluence

### PR DESCRIPTION
**Pull request by marsmensch**
atlassian-confluence 5.6.5 (new formula)

### CODE REVIEW

File: Library/Formula/atlassian-confluence.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+class AtlassianConfluence < Formula
+  homepage "https://www.atlassian.com/software/confluence/"
+  url "http://www.atlassian.com/software/confluence/downloads/binary/atlassian-confluence-5.6.5.tar.gz"
+  sha1 "4999940f0b0837961d72310128d8376c4025c504"
+
+  depends_on :java => "1.7"
+
+  def install
+    # TODO
```
Comment (mistydemeo) [inline Library/Formula/atlassian-confluence.rb]: Is there still anything `TODO` here?

**Comment by adamv** [comment]
Possible candidate for https://github.com/atlassian/homebrew-tap ?

**Comment by mikemcquaid** [comment]
What's the license for this?

File: Library/Formula/atlassian-confluence.rb (hunk 1 above)
Comment (marsmensch) [inline Library/Formula/atlassian-confluence.rb]: Sorry! No, i forgot to remove this line and brew audit did not catch it
either. Tested and works.
Am 06.01.2015 18:46 schrieb "Misty De Meo" <notifications@github.com>:

**Comment by marsmensch** [comment]
@mikemcquaid for the formula or confluence? wasn't aware of the tape, but since bamboo has also just been merged i assumed it would be ok.

**Comment by mikemcquaid** [comment]
For confluence.

**Comment by marsmensch** [comment]
@mikemcquaid proprietary (mostly), just as jia and bamboo https://www.atlassian.com/licensing/confluence this can be used for free by oss projects and edu, but a license must be obtained. free eval 30 days for all.

**Comment by marsmensch** [comment]
@jacknagel @mikemcquaid same as for atlassian-jira. Please let me know if i should fix the remaining issues in the Formula. If you don't want these, i will probably re-create them as Casks.

**Pull request closed without merging**